Content.Shared/Localizations/ContentLocalizationManager.cs

```diff
@@ -147,8 +147,9 @@ public static string FormatDirection(Direction dir)
         /// </summary>
         public static string FormatPlaytime(TimeSpan time)
         {
+            time = TimeSpan.FromSeconds(Math.Ceiling(time.TotalSeconds));
```

Comment:
```suggestion
            time = TimeSpan.FromSeconds(Math.Ceiling(time.TotalMinutes));
```
with the caveat that I have no idea if there is a `TotalMinutes` field.

This implementationshould round up no matter what, meaning it'll be
- 1 minute at 0.01 seconds
- 60 minutes at 59.1 seconds (should format as `1H 0M`)
- 61 minutes at 60.1 seconds (should format as `1H 1M`)

Reply:
oop, yeah, edited.
That'd make for a very incorrect result.

Basically the idea is `Timespan -> raw minutes -> Timespan`, so yeah, it absolutely needs to be minutes on both sides.

Reply:
I shall make thy requested change.